src/core/_modules/global/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "dns" {
```

Comment:
What is this?

Reply:
Since "dns" is a submodule of "global", it is necessary to export the outputs of the dns resources and access them like this:
`module.global.dns.private_dns_zones`

I tried to access the module directly with something like `module.global.module.dns.private_dns_zones`, but with no luck. Also it would look that good.